tasks/compile-stylesheets.mjs

```diff
@@ -66,6 +66,9 @@ export async function compileStylesheet ([modulePath, { srcPath, destPath }]) {
       file: moduleSrcPath,
       outFile: moduleDestPath,
 
+      // Turn off dependency warnings
+      quietDeps: true,
```

Comment:
Whilst this PR fixes `govuk-frontend` deprecation warnings we add `quietDeps: true` to ignore:

```console
govuk_frontend_toolkit
govuk-elements-sass
```

See https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#silence-deprecation-warnings-from-dependencies-in-dart-sass